### PR TITLE
Gp2 365 target market service page

### PR DESCRIPTION
**Pull request by rafa-garcia**
- [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.

### CODE REVIEW

File: core/models.py [hunk 1]
```diff
@@ -239,6 +239,38 @@ class LandingPage(CMSGenericPage):
     ]
 
 
+class ExtendedLandingPage(CMSGenericPage):
```
Comment (bobmeredith) [inline core/models.py:242]: So I just tried adding the generic template to the original landing page, and AFAICT, it works just fine.
The landing page is already in the core app - so it doesn't look like we need this new page at all - merely add the template and include it in the list of templates available in the landing page model and bingo.
`    template_choices = (
        ('learn/landing_page.html', 'Learn'),
        ('core/extended_landing_page.html', 'Generic'),
    )`
The only thing that's slightly naff is the name of the page - it should possibly be more like 'generic page' than 'Landing page' but heh ho!

**Comment by bobmeredith** [comment]
@rafa-garcia Could you leave a comment when you've reviewed this - as you probably can't review yourself.

File: core/models.py [hunk 2]
```diff
@@ -229,13 +230,26 @@ class LandingPage(CMSGenericPage):
         related_name='+'
     )
 
+    body = StreamField([
+        ('section', core_blocks.SectionBlock()),
+        ('title', core_blocks.TitleBlock()),
+        ('text', blocks.RichTextBlock(icon='openquote', helptext='Add a textblock')),
+        ('image', core_blocks.ImageBlock()),
+    ], null=True, blank=True)
```
Comment (webbyfox) [inline core/models.py:238]: This will make body field optional. Is this expected?

Comment (rafa-garcia) [reply core/models.py:238]: It is expected, this image(s) is not to be mistaken for the page's main image

**Comment by rafa-garcia** [comment]
Looks great @bobmeredith 👍

File: core/models.py (hunk 2 above)
Comment (bobmeredith) [reply core/models.py:238]: Yes, optional is good - this is the landing page so it could just have the title, description and a button.

**Pull request merged**